Template10 (Library)/Services/NavigationService/INavigationService.cs

```diff
@@ -23,6 +23,7 @@ public interface INavigationService
 
         void Refresh();
 
+        [Obsolete("Use Template10.Services.ViewService.IViewService.OpenAsync instead")]
```

Comment:
Let's consider a way to leave this alone. Could we change the signature to return an instance of IViewService or something to encapsulate the use of the service more simply for developers using T10?